src/routes/load/Load.test.tsx

```diff
@@ -22,8 +23,10 @@ describe('<Load>', () => {
 
     render(<Load />, customState)
 
-    expect(screen.getByText('Add existing Safe', { selector: 'h2' })).toBeInTheDocument()
-    expect(screen.getByTestId('load-safe-form')).toBeInTheDocument()
+    await waitFor(() => {
```

Comment:
Unless you are waiting for something to load inside the rendering, you should replace this here with act. Any interaction with the UI will display the act warning in the console.

Reply:
the only thing that worked for me its to wrap this asserts inside of a `await waitFor(() => { ... })`
if I use `act(() => {... })` instead, I still get the annoying warnings in the console 😢